### DISCUSSION TITLE
Bot suddenly Stops and heroku account gets ban

### DISCUSSION THREAD

**Discussion by SoloLeveling7seven** [The-Hunter-Oraganization/Erika-Amano#2] [General]
![Screenshot_2022-10-08-07-00-49-004_com android chrome](https://user-images.githubusercontent.com/109235836/194681084-a44047a6-f7c3-4233-b488-d4e198bc36ee.jpg)

**Comment by soheru** [comment]
Hello,

Thank you for reaching out and bringing this issue to our attention. We apologize for the inconvenience caused by the bot suddenly stopping and your Heroku account being banned.

It seems that the bot's behavior violated Heroku's policies, resulting in the account being blocked. Unfortunately, there is nothing we can do from our end to rectify this situation. We apologize for any delays in our response and any difficulties this may have caused.

We recommend reviewing Heroku's policies and guidelines to ensure compliance with their terms of service. If you have any further questions or concerns, please let us know, and we'll do our best to assist you within the limits of our capabilities.

Thank you for your understanding.

Best regards,
Soheru
